pkg/kubelet/images/image_manager.go

```diff
@@ -121,6 +121,13 @@ func (m *imageManager) EnsureImageExists(ctx context.Context, pod *v1.Pod, conta
 		})
 	}
 
+	// We append the runtime handler annotation to the end so that it overwrites any previous
+	// value that might have been set by the pod config. The array will be converted into a map
+	// by toRuntimeAPIImageSpec before calling the image service.
+	podAnnotations = append(podAnnotations,
+		kubecontainer.Annotation{Name: "kubernetes.io/runtimehandler", Value: runtimeHandler},
```

Comment:
can this be defined somewhere public so CRI implementations can import the variable? maybe in staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/constants.go

Reply:
Thanks for the suggestion. 

Done in the latest version.